index.html

```diff
@@ -1615,6 +1618,179 @@ <h3>
         <a>related applications</a> instead.
       </p>
     </section>
+    <section>
+      <h3>
+        `scope_extensions` member
+      </h3>
+      <p>
+        Each <dfn>scope extension</dfn> extends the [=manifest/navigation scope
+        of a manifest=] by describing [=URLs=] that should be treated as being 
+        [=manifest/within scope=]. [=Scope extensions=] can add URLs to scope 
+        but not remove them.
```

Comment:
We can probably remove the last sentence and add 'additional'?
```suggestion
        Each <dfn>scope extension</dfn> extends the [=manifest/navigation scope
        of a manifest=] by describing additional [=URLs=] that should be treated as
        being [=manifest/within scope=].
```